### ISSUE TITLE
Not enough puns

### ISSUE THREAD

**Issue by eddieantonio** [AdasTeam/AdaBotSlack#2]
I mean, there are none! It's basically a pun-ishable offence! The entry in the array may as well be `null` and `(void 0)`!

**Comment by kpatenio** [comment]
@eddieantonio yes, you're right. This is an issue that must be addressed immediately!
